source/install/install-ubuntu-1604-postgresql.rst

```diff
@@ -100,5 +100,9 @@ Assume that the IP address of this server is 10.10.10.1
       You might have to install the PostgreSQL client software to use the command.
 
   The PostgreSQL interactive terminal starts. To exit the PostgreSQL interactive terminal, type ``\q`` and press **Enter**.
+  
+13. (Optional) Postgres recommends the standard way to pass passwords is through a ``.pgpass`` file. It is a special file that HAS to be 0600 in permissions otherwise it doesnt work. One can also pass the location of the file using the ``PGPASS`` env variable.
```

Comment:
As far as I understand using `.pgpass` is just a conveniency to remove the need to type the password at every login. Taking a quick look at the [official documentation](https://www.postgresql.org/docs/9.5/libpq-pgpass.html) it doesn't appear to be explicitly recommended nor a standard way to authenticate. Also the environment variable to pass would be `PGPASSFILE`.

Reply:
I've edited the content a bit to work with @streamer45's recommendations. @fakela, let me know what you think. If you're happy with the changes, you can commit them. Otherwise, more than happy to talk them over. :)

Reply:
Looks alot simplified, I will commit them.
 Also are we adding this content on the same section of the other OS's. If yes I would open a seperate PR for them.